### PR TITLE
(major) Upgrade to Polymer 3

### LINKED ISSUE #12: Upgrade to Polymer 3
Dependencies:

* [x] cosmoz-tree https://github.com/Neovici/cosmoz-tree/issues/20 https://github.com/Neovici/cosmoz-tree/pull/21

### PR DESCRIPTION
**Pull request by cristinecula**
Fixes #12

### CODE REVIEW

**Comment by cristinecula** [comment]
Depends on:

```
"@neovici/cosmoz-tree": "plumelo/cosmoz-tree#NEOV-377-polymer-3",
```

**Comment by cristinecula** [comment]
@megheaiulian @nomego

File: cosmoz-treenode.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+		if (keyProperty === 'pathLocator') {
+			path = ownerTree.getPathNodes(keyValue);
+		} else {
+			node = ownerTree.getNodeByProperty(keyValue, keyProperty);
+			if (node && node.pathLocator) {
+				path = ownerTree.getPathNodes(node.pathLocator);
+			}
+		}
+		if (Array.isArray(path) && path.length) {
+			for (i = path.length - 1; i >= 0; i--) {
```
Comment (nomego) [inline cosmoz-treenode.js]:
```suggestion
			for (let i = path.length - 1; i >= 0; i--) {
```

File: cosmoz-treenode.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+			i;
+
+		if (!ownerTree || keyProperty == null || keyValue === undefined) {
+			return;
+		}
+
+		if (keyProperty === 'pathLocator') {
+			path = ownerTree.getPathNodes(keyValue);
+		} else {
+			node = ownerTree.getNodeByProperty(keyValue, keyProperty);
```
Comment (nomego) [inline cosmoz-treenode.js]:
```suggestion
			const node = ownerTree.getNodeByProperty(keyValue, keyProperty);
```

**Comment by cristinecula** [comment]
@nomego @Wurper ready for release!

**Comment by cristinecula** [comment]
@nomego build errored because of yarn cache

**Pull request merged**